### PR TITLE
Updating rails version to 5.2.4.1

### PR DESCRIPTION
**Pull request by clivecorbishley**
Zadev rails version is dependant on this gem. So we should have this ready before we bump zadev

### CODE REVIEW

File: integration_test_kit.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md']
 
-  spec.add_dependency 'rails', '~> 5.1.6'
+  spec.add_dependency 'rails', '~> 5.2.4'
```
Comment (EthanMarrs) [inline integration_test_kit.gemspec]: @clivecorbishley Couldn't we do a greater than? Also, if we change this we'll need to bundle and release a new version.

Comment (EthanMarrs) [reply integration_test_kit.gemspec]: Also, only test is dependent on IntegrationTestKit, not prod.

Comment (clivecorbishley) [reply integration_test_kit.gemspec]: I'm not sure what effects Rails 6 could have here, but sure could try.

Comment (EthanMarrs) [reply integration_test_kit.gemspec]: @clivecorbishley Yeah, I guess it's quite possible Rails 6 could break things. Don't think any of the engine stuff has been touched though

Comment (clivecorbishley) [reply integration_test_kit.gemspec]: @EthanMarrs Remember this? 😄 Got reminded again by bundle that our Rails version is still locked in... Have updated it now

Comment (clivecorbishley) [reply integration_test_kit.gemspec]: I am going to test it out on a branch on zadev

**Comment by EthanMarrs** [comment]
@clivecorbishley You're going to have to release a new version on RubyGems too

**Pull request closed without merging**